### ISSUE TITLE
Websocket Rails crashes RoR server bc of redis conflicts

### ISSUE THREAD

**Issue by Tectract** [websocket-rails/websocket-rails#376]
see here:

https://github.com/websocket-rails/websocket-rails/issues/191#issuecomment-150484131

not sure why that was closed, it's a rather critical and still unresolved bug for me. Is there some workaround for these symphony/redis related crashes that doesn't involve forking the project and/or modifying the source? I found that by commenting a single include line I could work around this issue, so maybe that include line could just be conditional or something.

**Comment by depili** [comment]
Yes, that bug is still around. My recommendation is to abandon websocket-rails as it isn't really maintained and has many bugs and use tubesock instead. Just rewrote my projects websocket-protocol on top of tubesocks.
